database/fixtures.sql

```diff
@@ -3,15 +3,76 @@ BEGIN;
 
 INSERT INTO "users"(username, email,password)
 VALUES
-  ('lee','lee@yahoo.com','lee'),
-  ('emma','emma@yahoo.com','emma'),
-  ('cate','cate@yahoo.com','cate'),
-  ('steve','steve@yahoo.com','steve');
+  ('DanielEineving','daniel@eineving.se','danieleineving'),
+  ('JimmyBerlin','berlin.jimmy@gmail.com','jimmyberlin'),
+  ('MikaelLonn','mickelonn93@gmail.com','mikaellonn'),
+  ('ChristopherAkersten','christopher.akersten@gmail.com','christopherakersten'),
+  ('JohanAndersson','lol2kpe@gmail.com','johanandersson');
 
 INSERT INTO token(token, userid)
-values('testtoken',(select id from "users" where username = 'lee' LIMIT 1));
+values('testtoken',(select id from "users" where username = 'ChristopherAkersten' LIMIT 1));
 
-INSERT INTO customer (name, email) VALUES ('test', 'test');
-INSERT INTO customer (name, email) VALUES ('jhon doe', 'example@example.com');
+INSERT INTO customer (name, email) VALUES ('Testbolag', 'test@testbolag.se');
+INSERT INTO customer (name, email) VALUES ('Chalmers n Stuff', 'gangsters@tjalmers.se');
+
+INSERT INTO "order"(customerID, date) VALUES
+  ((select id from customer where name = 'Testbolag'), 2017-10-10),
+  ((select id from customer where name = 'Testbolag'), 2017-11-01),
+  ((select id from customer where name = 'Chalmers n Stuff'), 2017-09-09);
+
+INSERT INTO digitalpart (customerID, name) VALUES
+  ((select id from customer where name = 'Testbolag'), 'Screw'),
+  ((select id from customer where name = 'Testbolag'), 'Hinge'),
+  ((select id from customer where name = 'Chalmers n Stuff'), 'Emil'),
+  ((select id from customer where name = 'Chalmers n Stuff'), 'Emilia');
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)